circle.html

```diff
@@ -1,38 +1,54 @@
 <!DOCTYPE>
 <html>
+
 <head>
-	<title>Primitive Paint</title>
-	<link rel="stylesheet" type="text/css" href="assets/css/paper.css">
-	<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/paper.js/0.12.2/paper-full.min.js"></script>
-	<script type="text/paperscript" canvas="myCanvas">
+  <title>Primitive Paint</title>
+  <link rel="stylesheet" type="text/css" href="assets/css/paper.css">
+  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/paper.js/0.12.2/paper-full.min.js">
+  </script>
+  <script type="text/paperscript" canvas="myCanvas">
     var path;
     tool.onMouseDown = function(event) {
+      var BrushColor="#000000"
```

Comment:
the variable  `BrushColor` is being used in both `onMouseDown` and `onMouseDrag` so It should be declared globally instead of declaring it in every function.